### PR TITLE
added method saveBtn into methods.js

### PR DESCRIPTION
**Pull request by OksanaK-sys**
added steps into entityReferenceValues.js

### CODE REVIEW

**Comment by OksanaK-sys** [comment]
> One more thing: after this command - **$(selectors.navigationBar.sideBar).click();** - you always go to this page - https://ref2.eteam.work/index.php?action=screen&screen_id=43&filter&mod=2

File: helper/methods.js [hunk 1]
```diff
@@ -128,3 +128,7 @@ export function scrollLeftAndClick(selector) {
     selector.scrollIntoView({inline: "end"});
     selector.click();
 }
+
+export function saveBtn() {
+    $(selectors.common.saveBtn).click();
+}
```
[the remark this answers is not in the document]
Comment (OksanaK-sys) [reply helper/methods.js]: I changed everything, could you check pls?

File: test/specs/entityReferenceValues.js [hunk 2]
```diff
@@ -42,10 +50,63 @@ describe('REFERENCE VALUES', () => {
...
+    });
+    it('11/RV-Should verify record restored as Draft', () => {
+        $(selectors.recycleBin.recycleBinRestoreAsDraft).click();
+        expect($(selectors.common.entityCarBody).getText()).to
+            .equal(expected.recycleBin.expectedEmptyBinText);
+        $(selectors.referenceValues.referenceValuesMenu)
+            .scrollIntoView();
+        $(selectors.referenceValues.referenceValuesMenu).click();
+        expect(getRowCount()).to.equal(expected.common.oneRecord);
+        expect($(selectors.common.checkSquare).isDisplayed());
```
Comment (Burdishka) [inline test/specs/entityReferenceValues.js:101]: This verification doesn't work. This element is not displayed. You have a **draft** record - **"draftPencil": ".fa.fa-pencil"**

if you add true at the end expect($(selectors.common.checkSquare).isDisplayed()).true; - your test would fell

Comment (OksanaK-sys) [reply test/specs/entityReferenceValues.js]: fixed, thank you

**Review by Burdishka — APPROVED** [review]

**Pull request merged**